app/src/main/java/com/projectthrive/chess/ui/main/MainFragment.kt

```diff
@@ -24,11 +25,20 @@ class MainFragment : Fragment() {
         val rootView = inflater.inflate(R.layout.main_fragment, container, false)
         val mainBoard = rootView.findViewById<GridLayout>(R.id.main_board)
 
+        viewModel = GameModel()
```

Comment:
We have a small issue here.
We're missing the `Model` of our MVVM Architecture.

We have a view: `MainFragment`
We have a ViewModel: `GameModel`
But we do not have a model yet.